webpack.config.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
- [ ] I noticed that you disabled **Eslint** . Please remove the **eslint disable** statement and try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1:

Reply:
I have done all I can. This Eslint error wont go away. It passes on VSCODE but then fails on github after push. Can you help?
Thanks.

Reply:
- [ ] Please, this needs to be fixed. Look at the error message on GitHub when the check fails and look up what needs to be done through the documentation, StackOverflow or even start up a zoom call with me. I will be happy to help. This is how we learn. 🤝